deploy/manifests/base/assigner/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         - name: assigner
           image: storetheindex
           command: ["/sbin/tini", "--", "/usr/local/bin/start_assigner"]
+          args: ["daemon"]
```

Comment:
Though valid YAML, why use JSON syntax to specify arrays when one can use `-` which is consistent with how arrays are specified across the manifest?

Reply:
fixed